data/data/aws/bootstrap/main.tf

```diff
@@ -249,3 +249,11 @@ resource "aws_security_group_rule" "bootstrap_journald_gateway" {
   from_port   = 19531
   to_port     = 19531
 }
+
```

Comment:
There are a lot of terraform changes. With the removal of terraform and addition of cluster api should we consider moving this over cluster api after the base installs work?

Reply:
@barbacbd yes, cluster API work is already in progress. The plan is to deliver both changes: https://issues.redhat.com/browse/OCPSTRAT-1154